README.md

```diff
@@ -35,59 +43,74 @@ The MixedReality-SpectatorView repository uses Unity packages, git submodules an
 
-Depending on what release you are using the correct setup script may vary. Choose the appropriate script below based on the git branch that you have checked out in your clone of the MixedReality-SpectatorView repository:
+Depending on what release you are using the correct setup script may vary. Choose the appropriate script below based on the git branch that you have checked out in your clone of the MixedReality-SpectatorView repository.
 
 #### Setting up the `release/1.0.0-beta` branch
 
 If you are using the release/1.0.0-beta branch, You will need to run the following command:
 
-1. Run `'tools/Scripts/ResetSamples.bat'` as an administrator on a PC  (On Mac or Linux, you can run `'sh /tools/scripts/ResetSamples.sh'`).
+1. Run `'tools/Scripts/ResetSamples.bat'` as an administrator on a PC  (On Mac or Linux, you can run `'sh /tools/scripts/ResetSamples.sh'`). These scripts are located within your MixedReality-SpectatorView submodule.
 
 #### Setting up the `master` branch
 
 If you are using the master branch, you will need to run the following command:
 
-1. Run `'tools/Scripts/SetupRepository.bat'` as an administrator on your PC (On Mac or Linux, you can run `'sh /tools/scripts/SetupRepository.sh'`).
-
+1. Run `'tools/Scripts/SetupRepository.bat'` as an administrator on your PC (On Mac or Linux, you can run `'sh /tools/scripts/SetupRepository.sh'`). These scripts are located within your MixedReality-SpectatorView submodule.
 
+![Marker](doc/images/SetupRepo.png)
 
 ### Samples
 
 After going through the setup steps in 'Obtaining the code' and 'Setting up your local environment', sample projects will be configured for use in your clone of the MixedReality-SpectatorView repository. It's easy to get started by building off one of the samples or by inspecting them to understand project setup. For more information, see [Samples](samples/README.md).
 
-## Setting up your own project
-
 ### Adding references to your own project
 
 After obtaining a local clone of the MixedReality-SpectatorView repository and resolving its external dependencies (see above), the suggested mechanism for referencing the code is by adding symbolic linked directories to your Unity project's Assets folder. You can do this with the following:
 
+> Note: Symbolic linked directories should be setup as relative paths. Using relative paths should allow directories to resolve correctly regardless of where you or your team members clone your project repository in their local file systems. The instructions below demonstrate setting up symbolic linked directories based on the following paths:
+>* **Project repository directory:** c:\Your\Unity\Project
+>* **Project Assets directory:** c:\Your\Unity\Project\Assets
+>* **MixedReality-SpectatorView submodule directory:** c:\Your\Unity\Project\sv

```

Comment:
>sv [](start = 77, length = 2)

Are we recommending "sv" as a name because of path limits?

Reply:
Yeah, that's the main reason.